lib/pronto/version.rb

```diff
@@ -0,0 +1,5 @@
+module Pronto
+  module RailsMigrations
```

Comment:
This is not entirely cool either because the actual `Pronto::RailsMigrations` is a class. Maybe just put `VERSION` inside of the actual class.

If for some reason both version file and actual gem are required then this will break.